src/app/components/students/students.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit, Input } from "@angular/core";
+
+import { Student, Button } from "../../common/entities";
+
+import { DbService } from "../../common/services/db.service";
+
+@Component({
+  selector: "app-students",
+  templateUrl: "./students.component.html",
+  styleUrls: ["./students.component.scss"]
+})
+export class StudentsComponent implements OnInit {
+
+  private button: Button = {
+    text: "+",
+    class: "btn--add"
+  };
+
+  private formInfo: any = {
```

Comment:
spread these fields in `html` file. It gives you more conveniency to change if you put it in html